scalanet/ut/src/io/iohk/scalanet/codec/FramingCodecSpec.scala

```diff
@@ -23,7 +23,7 @@ class FramingCodecSpec extends FlatSpec with EitherValues {
 
     forAll(genString()) { listOfString =>
       val enc = listOfString.map(encoder.encode(_).require).foldLeft(BitVector.empty)((acc, bit) => acc ++ bit)
-      val decoded = encoder.streamDecode(enc).right.value
+      val decoded = encoder.streamDecode(enc).toOption.get
```

Comment:
Sorry for that (and for `.swap.toOption.get` as well) :(
I found no alternatives to fit both 2.12 and 2.13 `Either` methods without triggering any compiler warning.

Reply:
What was the problem here?

Reply:
`.right.value` produces a warning in scala 2.13 because `Either` became right-based.
We could probably disable fatal-warnings directive but it will make the codebase even worse.
I wouldn't try doing some ad-hoc methods for `Either`
Or even separate sources for 2.12 and 2.13.

Reply:
Gotcha, thanks.